priv/code_navigation/src/diagnostics.new.erl

```diff
@@ -0,0 +1,7 @@
+-module(diagnostics).
```

Comment:
I would rename the `.new` in the filename into `_new` (also below).

Reply:
I used the `.` style to indicate that it is not a regular file, but is a different version of the `diagnostics.el` file,  which is used as input to the test for diff generation, between `diagnostics.erl` and `diagnostics.new.erl`.  So the module id should not change either.